src/pages/api/apiUtils/userData.ts

```diff
@@ -352,6 +355,32 @@ export const getPeriodMultipleServicesTicketJson = (
         };
     });
 
+    if (basePeriodTicketAttributes.type === 'multiOperator') {
```

Comment:
if type is undefined then would this handle it?

Reply:
I dont think the type can be undefined, we use the `getBasePeriodTicketAttributes` method in the other functions.